src/utils/dueDateFormatter.ts

```diff
@@ -0,0 +1,27 @@
+import dayjs from 'dayjs'
+
+export const DueDateFormatter = (date: string) => {
+  const parsedDate = dayjs(date)
+  const monthFormats: { [key: string]: string } = {
+    January: 'Jan',
+    February: 'Feb',
+    March: 'March',
+    April: 'April',
+    May: 'May',
+    June: 'June',
+    July: 'July',
+    August: 'Aug',
+    September: 'Sep',
+    October: 'Oct',
+    November: 'Nov',
+    December: 'Dec',
+  }
+
+  const month = parsedDate.format('MMMM')
+  const day = parsedDate.format('D')
+  const year = parsedDate.format('YYYY')
+
+  const formattedMonth = month.length <= 5 ? month : monthFormats[month]
+
+  return `${formattedMonth} ${day}, ${year}`
```

Comment:
We already have something exactly like this in `DueDateLayout`.

Let's remove this code from that file and implement this function there and keep our code clean.